Objects/Converters/ConverterTeklaStructures/ConverterTeklaStructuresShared/PartialClasses/ConvertDirectShapeMesh.cs

```diff
@@ -48,23 +74,33 @@ public void MeshToNative(Base @object, List<GE.Mesh> displayValues)
           shapeName = matchingShape.Name;
           result = true;
         }
+        else
```

Comment:
I'd rather add a ticket labelled with Tech debt and reference that here than add more TODOs.